docs/gethue/content/en/posts/2021-05-18-installing-hue-slack-app-in-three-simple-steps.md

```diff
@@ -0,0 +1,96 @@
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.10
+
+---
+Ever wondered about having a SQL Assistant which can help you to collaborate better with other SQL users in your team? That too in Slack itself!?
+
+This Slack Integration for Hue can fit those shoes for assisting you with your SQL queries in your Slack channels. [Easy to use](https://docs.gethue.com/user/concept/#slack) and now even easy to install for Slack workspace admins in just 3 simple steps!
+
+Kudos to Slack for recently rolling out its [App Manifest](https://api.slack.com/reference/manifests) beta feature for Slack Apps! With a manifest now, you can create an app with a predefined configuration.
+
+The latest version of the App Manifest is checked-in the [Hue Repository](https://github.com/cloudera/hue/blob/master/tools/slack/manifest.yml).
+
+In the shared manifest, Update the two _demo.gethue.com_ with **your Hue instance hostname:**
+- Under **unfurl_domains**
+- Under **event_subscriptions**, in **request_url** `https://<hue-instance-hostname>/desktop/slack/events/`
+
+Now, it's time to create your own app:
+
+1. Go to https://api.slack.com/apps and click **Create New App**.
+2. Choose **From an app manifest** option and workspace that you'll be installing the app and click **Next**.
+3. Choose **YAML** and paste the Manifest code (make sure you do the necessary changes mentioned above) and click **Next**.
+4. Read the review summary and if everything’s correct, click **Create**.
+5. Once the app is created, install it in the workspace!
+
+Completing the last step of plugging it with Hue, it's time to update your hue.ini configuration file:
+
+6. Go to the **OAuth & Permissions page**, copy the **Bot User OAuth Token** and update **slack_bot_user_token** (e.g. xoxb-xxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxx)
+7. Similarly, go to the **Basic Information** page, copy the **Verification Token** and update **slack_verification_token**
+8. Paste this in your hue.ini file under `[desktop]` section
+
+        [[slack]]
+        is_enabled=true
+        slack_verification_token=<your-slack-verification-token>
+        slack_bot_user_token=<your-slack-bot-user-token>
+
+That’s it! Your own Hue App is ready to roll!
+
+Read its [user guide](https://docs.gethue.com/user/concept/#share-to-slack) or the [blog](https://gethue.com/blog/2021-04-09-collaborate-on-your-sql-queries-and-results-directly-within-slack/) and stay tuned for upcoming updates.
+
+### Try it out!
+
+Get your hands on demo Hue App by:
+
+Running some queries on the [demo live editor](https://demo.gethue.com/).
+
+Joining the [demo Slack workspace](https://join.slack.com/t/hue-bot-dev/shared_invite/zt-pqep28s6-zd69W73EOWL5pCVDz06OBg) to see the app in action or by using the following credentials:
```

Comment:
FYI this is for creating an account for me..

Reply:
No need to create
1. Signout from the workspace from @romainr account
2. Join via this link and give demo credentials

I think this will work for you as I was facing same problem and now I am signed in as the @Demo User in the demo-gethue

Reply:
This way we can also see if demo user can be signed in from multiple places to or not ;)

Reply:
Thats strange, looking into it for a better tryout process